src/components/Subjects/Picture.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import styles from './SubjectStyle.scss';
+
+export default function Picture() {
+
+  return (
+    <div className={styles.subjectImage}>
+      <img src='https://liberal-international.org/wp-content/uploads/2017/06/500px-Pink_Venus_symbol.svg_.png' class="boom" width= "150px" height="150px"/>
```

Comment:
this also has a `class` attribute that seems like it needs to be switched to `className`. are you checking the console or your terminal for errors? there should have been an error pointing this out